### PR TITLE
chore(node): enable `parallel/test-whatwg-*` tests

### PR DESCRIPTION
**Pull request by iuioiua**
This PR enables some `parallel/test-whatwg-*` tests and `parallel/test-btoa-atob.js`.

### CODE REVIEW

File: node/_tools/config.json [hunk 1]
```diff
@@ -692,7 +693,25 @@
       "test-util.js",
       "test-vm-static-this.js",
       "test-webcrypto-sign-verify.js",
-      "test-whatwg-url-properties.js",
```
Comment (kt3k) [inline node/_tools/config.json:695]: Is this removal intentional?

Comment (iuioiua) [reply node/_tools/config.json:695]: Whoops! Nope, not intentional. Fixed now. Thanks.

**Review by kt3k — APPROVED** [review]
LGTM. Thanks!
